project/Dependencies.scala

```diff
@@ -7,13 +7,12 @@ object Dependencies {
   val scala210Version = "2.10.4"
   val scala211Version = "2.11.5"
 
-  val picklingVersion = "0.10.0"
+  val picklingVersion = "0.10.1"
   val pickling210 = "org.scala-lang.modules" % "scala-pickling_2.10" % picklingVersion
   val pickling211 = "org.scala-lang.modules" % "scala-pickling_2.11" % picklingVersion
   val pickling = "org.scala-lang.modules" %% "scala-pickling" % picklingVersion
 
   private val jsonTuples = Seq(
-    ("org.json4s", "json4s-core", "3.2.10"),
```

Comment:
It's probably more DRY to use the whatever AST version `"org.spire-math", "json4s-support", "0.6.0"` brings in.
In the future if we can get super stable AST JAR going, and we change the version number of jawn to 0.8.0 or whatever, we might forget to update/remove dependencies to json4s-ast.